tests/config/nextflow.config

```diff
@@ -1,20 +1,15 @@
-
 params {
     outdir = "output/"
     publish_dir_mode = "copy"
     enable_conda = false
```

Comment:
```suggestion
    enable_conda = false
    singularity_pull_docker_container = false
```

Reply:
Do you want me to re-enable singularity?